server/lib/packets/packet.go

```diff
@@ -4,9 +4,10 @@ import (
 	"log"
 	"fmt"
 
-	"../assets"
-	"../models"
-	proto "../../proto"
+	"github.com/umegaya/yue"
+	"github.com/umegaya/channer/server/lib/assets"
+	"github.com/umegaya/channer/server/lib/models"
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)